doc/classes/Node2D.xml

```diff
@@ -92,7 +92,7 @@
 			<argument index="0" name="local_point" type="Vector2">
 			</argument>
 			<description>
-				Converts a local point's coordinates to global coordinates.
+				Transforms the provided local position into a position in global coordinate space. The input must be local relative to the [Node2D] it is called on. i.e. Applying this method to the positions of child nodes will correctly transform their positions into the global coordinate space, but applying it to a node's own position will give an incorrect result, as it will incorporate the node's own transformation into its global position.
```

Comment:
`must be [local relative to]` could also be written as `is` or `is expected to be [local relative to]`, for a slightly less assertive sentence.

`i.e.` ("in other words") should be `e.g.` ("for example"), as a certain friendly french person has taught me over the years.

Likewise for `to_local`.

Reply:
Good ideas. Made those changes.